### PR TITLE
Fix `doc` in Colab

### LINKED ISSUE #1660: doc() does not work in Colab
**Describe the bug**
The help method doc() does not work in colab. On the other hand ?? is working to show the source code.

**Provide your installation details**

```text
=== Software === 
python        : 3.6.7
fastai        : 1.0.45
fastprogress  : 0.1.19
torch         : 1.0.1.post2
nvidia driver : 410.79
torch cuda    : 10.0.130 / is available
torch cudnn   : 7402 / is enabled

=== Hardware === 
nvidia gpus   : 1
torch devices : 1
  - gpu0      : 11441MB | Tesla K80

=== Environment === 
platform      : Linux-4.14.79+-x86_64-with-Ubuntu-18.04-bionic
distro        : #1 SMP Wed Dec 19 21:19:13 PST 2018
conda env     : Unknown
python        : /usr/bin/python3
sys.path      : 
/env/python
/usr/lib/python36.zip
/usr/lib/python3.6
/usr/lib/python3.6/lib-dynload
/usr/local/lib/python3.6/dist-packages
/usr/lib/python3/dist-packages
/usr/local/lib/python3.6/dist-packages/IPython/extensions
/root/.ipython
```

**To Reproduce**
Just open a colab notebook, paste and evaluate the following code:
```
from fastai import *
from fastai.vision import *
doc(accuracy)
```

**Expected behavior**
A summary of the function with a link to the source, like in the videos.

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by ericricky**
Fixes #1660.

### CODE REVIEW

**Comment by sgugger** [comment]
This is the good fix AFAICT (I'll trust you for the `in_collab` test). It doesn't change the current behavior in jupyter notebook and fixes it in collab (see discussion in the mentioned issue). @bearpelican, let me know if you feel differently.

File: fastai/utils/ipython.py [hunk 1]
```diff
@@ -16,6 +16,13 @@ def is_in_ipython():
 
 IS_IN_IPYTHON = is_in_ipython()
 
+def is_in_colab():
+    "Is the code running in Google Colaboratory?"
+    if IS_IN_IPYTHON and os.path.exists('/var/colab'): return True
```
Comment (stas00) [inline fastai/utils/ipython.py]: Hoping nobody happens to have this path for whatever other reason - perhaps not the best check.
Here https://stackoverflow.com/a/39662359/9201239 in comments someone proposed:

> If someone is interested in detecting whether the notebook is running on Google Colab you can check this: get_ipython().__class__.__module__ == "google.colab._shell" – guiferviz Nov 19 '18 at 22:04

Comment (ericricky) [reply fastai/utils/ipython.py]: Yes much better, thank you!

**Comment by stas00** [comment]
Thanks for that change, @ericricky. But I'll let @bearpelican to review.

The proposed change makes it so that `get_ipython().run_cell_magic()` will be run only in colab, which I'm not sure at all is correct. But I didn't write that part, so Andrew will know for sure.

**Comment by sgugger** [comment]
I was the one to add it ;)
Thanks @ericricky  !

**Comment by ericricky** [comment]
@sgugger No problem, and thank you. Love the course and the library!

**Comment by bearpelican** [comment]
Awesome thanks for fixing @ericricky!

**Comment by ericricky** [comment]
@bearpelican No problem!

**Pull request merged**